components/openzwave-adapter/src/lib.rs

```diff
@@ -107,18 +107,18 @@ trait RangeChecker {
     fn should_send(&self, &Value, EventType) -> bool;
 }
 
-impl RangeChecker for Option<Box<Range>> {
+impl RangeChecker for Option<Value> {
     fn should_send(&self, value: &Value, event_type: EventType) -> bool {
         match *self {
             None => event_type == EventType::Enter, // no range means we send only Enter events
-            Some(ref range) => range.contains(value)
+            Some(ref range) => range == value // FIXME: This won't scale up to interesting ranges.
         }
     }
 }
 
-impl RangeChecker for Range {
+impl RangeChecker for Value {
     fn should_send(&self, value: &Value, _: EventType) -> bool {
-        self.contains(value)
+        self == value // FIXME: This won't scale up to interesting ranges.
```

Comment:
what's the path forward with this FIXME? Maybe you could file an issue and put the issue # here and above ?